scripts/lib-franklin.js

```diff
@@ -428,6 +428,24 @@ export function buildBlock(blockName, content) {
   return (blockEl);
 }
 
+/**
+ * Gets the configuration for the given glock, and also passes
+ * the config throw all custom patching helpes added to the project.
+ *
+ * @param {Element} block The block element
+ * @returns {object} The block config (blockName, cssPath and jsPath)
```

Comment:
```suggestion
 * Gets the configuration for the given block, and also passes
 * the config through all custom patching helpers added to the project.
 *
 * @param {Element} block The block element
 * @returns {Object} The block config (blockName, cssPath and jsPath)
```

Reply:
🤦🏼 Thanks for catching those…
I don't think I ever made so many typos in just 3 lines 😓